CDP4IMEInstaller/Dependencies.wxs

```diff
@@ -8,9 +8,9 @@
 	<Fragment>
     <ComponentGroup Id="DEPENDENCIES" Directory="BIN">
 
-      <Component Id="comp_ExceptionReporter_WinForms" Guid="{DDBBAB53-5E15-4461-85BB-4B0456A090F6}" Location="local" Win64="$(var.Win64)">
-        <File Id="fil_ExceptionReporter_WinForms" Vital="yes" Source="$(var.CDP4IME-CE.TargetDir)ExceptionReporter.WinForms.dll" KeyPath="yes" />
-      </Component>
+	  <Component Id="comp_ExceptionReporter_WinForms" Guid="{DDBBAB53-5E15-4461-85BB-4B0456A090F6}" Location="local" Win64="$(var.Win64)">

```

Comment:
Are you sure that the old file will be removed when the new installer is installed on top of an existing installation? Otherwise I think a new GUID should be created and the Component Id should be renamed. Maybe even add a remove element as state in the wiki: https://github.com/RHEAGROUP/COMET-IME-Community-Edition/wiki/Installer

Reply:
in my earlier PR i created a new GUID for this

Reply:
It can be done either way. As is, the previous dependency will be overwritten. New entry with a new GUID can be added which will require a delete element for the old winforms dependency when replacing a previous install

Reply:
then i don't know what the best approach is and leave the decision to @SimonWoodRhea , i cannot advise